font/src/darwin/mod.rs

```diff
@@ -370,31 +428,14 @@ pub fn descriptors_for_family(family: &str) -> Vec<Descriptor> {
 
 impl Descriptor {
     /// Create a Font from this descriptor
-    pub fn to_font(&self, size: f64, load_fallbacks:bool) -> Font {
+    pub fn to_font(&self, size: f64, fallbacks:Option<Vec<Font>>) -> Font {
```

Comment:
Maybe this should be renamed `to_font_with_fallbacks` now that it can't be converted directly into a font. A constructor without the argument `to_font` could still be provided, and then the option wouldn't even be needed.